Userland/Libraries/LibPthread/pthread_cond.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright (c) 2021, Sergey Bugaev <bugaevc@serenityos.org>
+ * Copyright (c) 2019, Andreas Kling <kling@serenityos.org>
```

Comment:
I *think* these have traditionally gone earlier-to-later, I've only seen e.g. the 2019 entry come before the 2021 one